### DISCUSSION TITLE
[TEST] et [INSTALL] Thermostat Zigbee Moes BHT-002 - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#23] [Comments]
# [TEST] et [INSTALL] Thermostat Zigbee Moes BHT-002 - Haade.fr

test d’un thermostat zigbee par Moes le BHT-002

https://haade.fr/fr/blog/test-et-install-thermostat-zigbee-moes-bht-002

**Comment by cirtape** [comment]
Bonjour, pour quel genre de chauffage a tu tester le thermostat ? chauffage au sol électrique ou autre ?
Autre point, est 'il possible de changer les heures des différentes plages ? 
et d'après la datasheet, il garde en mémoire les consignes, donc meme après une coupure, le systeme est capable de chauffer comme il se doit la piece ? (pas de surchauffe avec une valeur de consigne élevée ? )
